proto-compiler/src/lib.rs

```diff
@@ -107,6 +107,14 @@ pub fn proto_compile() {
     let tenderdash_ver = tenderdash_version(tenderdash_dir);
 
     println!("[info] => Creating structs.");
+
+    #[cfg(feature = "grpc")]
+    tonic_build::configure()
+        .generate_default_stubs(true)
+        .compile_with_config(pb, &protos, &proto_includes_paths)
+        .unwrap();
+
+    #[cfg(not(feature = "grpc"))]
```

Comment:
If you need to both gRPC and socket server?

Reply:
For grpc, we compile using tonic (with client/server). If no grpc, we compile directly using prost (without client/server)